### PR TITLE
Add unit test for metric log

### PR DESCRIPTION
**Pull request by louyuting**
### Describe what this PR does / why we need it
contain file: aggregator.go, common.go

### Special notes for reviews

### CODE REVIEW

**Comment by sczyh30** [comment]
Could you please sign the CLA here: https://cla-assistant.io/alibaba/sentinel-golang?pullRequest=53

File: core/log/metric/aggregator.go [hunk 1]
```diff
@@ -119,6 +120,7 @@ func aggregateIntoMap(mm metricTimeMap, metrics map[uint64]*base.MetricItem, nod
 	}
 }
 
+// TODO Might the condition should be stricter here?
 func isActiveMetricItem(item *base.MetricItem) bool {
 	return item.PassQps > 0 || item.BlockQps > 0 || item.CompleteQps > 0 || item.ErrorQps > 0 ||
```
Comment (louyuting) [inline core/log/metric/aggregator.go]: Might the condition should be stricter here?
Such as: Resource,Classification,Timestamp.

**Comment by louyuting** [comment]
> Could you please sign the CLA here: https://cla-assistant.io/alibaba/sentinel-golang?pullRequest=53

Done,
Thanks~

File: core/log/metric/aggregator.go (hunk 1 above)
Comment (sczyh30) [reply core/log/metric/aggregator.go]: The function is designed for filtering "active" items (already assumed valid), while "active" means it has non-positive metric value.

Comment (louyuting) [reply core/log/metric/aggregator.go]: OK~thanks

**Comment by sczyh30** [comment]
Maybe it's better to have a unified path for test data (e.g. `test/data/...`)? Like:

- https://github.com/kubernetes/kubernetes/tree/master/test/e2e/testing-manifests
- https://github.com/istio/istio/tree/master/tests/testdata

**Comment by sczyh30** [comment]
Thanks!

**Pull request merged**